apps/web/src/services/alert-manager-v2/pages/ServiceDetailWebhookCreatePage.vue

```diff
@@ -1,9 +1,158 @@
+
+import { i18n } from '@/translations';
+
+import { useProperRouteLocation } from '@/common/composables/proper-route-location';
+
+import WebhookCreateForm from '@/services/alert-manager-v2/components/WebhookCreateForm.vue';
+import WebhookCreateSuccessMode from '@/services/alert-manager-v2/components/WebhookCreateSuccessMode.vue';
+import WebhookCreateTypeSelector from '@/services/alert-manager-v2/components/WebhookCreateTypeSelector.vue';
+import { SERVICE_DETAIL_TABS } from '@/services/alert-manager-v2/constants/alert-manager-constant';
+import { ALERT_MANAGER_V2_ROUTE } from '@/services/alert-manager-v2/routes/route-constant';
+import { useServiceCreateFormStore } from '@/services/alert-manager-v2/store/service-create-form-store';
+import type { createHeaderInfoByStep } from '@/services/alert-manager-v2/types/alert-manager-type';
+
+interface Props {
+    serviceId: string;
+}
+const props = withDefaults(defineProps<Props>(), {
+    serviceId: '',
+});
+
+const serviceCreateFormStore = useServiceCreateFormStore();
+const serviceCreateFormState = serviceCreateFormStore.state;
+
+const router = useRouter();
+
+const { getProperRouteLocation } = useProperRouteLocation();
+
+const storeState = reactive({
+    selectedWebhookTypeId: computed<string>(() => serviceCreateFormState.selectedWebhookType?.plugin_id || ''),
+    webhookName: computed<string>(() => serviceCreateFormState.webhookName || ''),
+});
+const state = reactive({
+    currentStep: 1,
+    isAllFormValid: computed<boolean>(() => {
+        if (state.currentStep === 1) return storeState.selectedWebhookTypeId !== '';
+        return storeState.webhookName !== '';
+    }),
+    headerInfo: computed<createHeaderInfoByStep>(() => {
+        if (state.currentStep === 1) {
+            return {
+                title: i18n.t('ALERT_MANAGER.SERVICE.INTEGRATE_TOOL_TITLE'),
+                desc: i18n.t('ALERT_MANAGER.SERVICE.INTEGRATE_TOOL_DESC'),
+            };
+        }
+        if (state.currentStep === 2) {
+            return {
+                title: i18n.t('ALERT_MANAGER.SERVICE.INTEGRATE_TOOL_TITLE'),
+            };
+        }
+        return {
+            title: i18n.t('ALERT_MANAGER.WEBHOOK.CREATE_SUCCESS_TITLE'),
+        };
+    }),
+});
+
+const handleClickCancelButton = () => {
+    router.push(getProperRouteLocation({
+        name: ALERT_MANAGER_V2_ROUTE.SERVICE.DETAIL._NAME,
+        params: {
+            serviceId: props.serviceId,
+        },
+        query: {
+            tab: SERVICE_DETAIL_TABS.WEBHOOK,
+        },
+    }));
+};
+const handlePrevNavigation = () => {
+    if (state.currentStep === 1) {
+        handleClickCancelButton();
+        return;
+    }
+
+    state.currentStep -= 1;
+};
+
+const handleActionButton = () => {
+    if (state.currentStep === 1) {
+        state.currentStep = 2;
+        return;
+    }
+    fetchCreateWebhook();
+};
+
+const fetchCreateWebhook = () => {
+    console.log('TODO: fetchCreateWebhook');
```

Comment:
Is it for comment?

Reply:
I left it on purpose to check after the API is out! I've marked it similarly in other files as well.